services/ui-src/src/utils/other/mask.test.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  convertToCommaSeparatedString,
+  isCustomMask,
+  isNumberStringMaskable,
+  maskValue,
+} from "utils";
+
+const commaSeparatedMaskAcceptableTestCases = [
+  { test: "0....0123", expected: "0.01" },
+  { test: "000000123", expected: "123.00" },
+  { test: "123", expected: "123.00" },
+  { test: "123.00", expected: "123.00" },
+  { test: ".05000000000000", expected: "0.05" },
+  { test: ".05", expected: "0.05" },
+  { test: ".5", expected: "0.50" },
+  { test: "0.05", expected: "0.05" },
+  { test: "1,234.00", expected: "1,234.00" },
+  { test: "1,234", expected: "1,234.00" },
+  { test: "1234", expected: "1,234.00" },
+  { test: "1,234.00", expected: "1,234.00" },
+  { test: "1,234", expected: "1,234.00" },
+  { test: "100,000,000", expected: "100,000,000.00" },
+  { test: "100000000", expected: "100,000,000.00" },
+  {
+    test: "Technically a wrong input that validation would pick up but would actually still work with 1 number",
+    expected: "1.00",
+  },
+];
+
+describe("Test isCustomMask", () => {
+  test("Check if good and bad mask values return accurately", () => {
+    const commaSeparated = isCustomMask("comma-separated");
+    const badMask = isCustomMask("cherry-tree");
+    expect(commaSeparated).toEqual(true);
+    expect(badMask).toEqual(false);
+  });
+});
+
+describe("Test isNumberStringMaskable", () => {
+  test("Check if strings with numbers are maskable", () => {
+    for (let testCase of commaSeparatedMaskAcceptableTestCases) {
+      expect(isNumberStringMaskable(testCase.test)).toEqual(true);
+    }
+  });
+
+  test("Check if strings without numbers are not maskable", () => {
```

Comment:
Suggestion for renaming:
```suggestion
  test("Non-numerical strings are rejected", () => {
```
or similar